<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server Util v1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>950</width>
      <height>650</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #f0f0f0, stop:1 #d0d0d0); /* Gradient background */
    border: 1px solid #ccc; /* Light border color */
    border-radius: 5px; /* Rounded corners */
    padding: 10px; /* Padding for content */
}</string>
    </property>
    <widget class="QPushButton" name="PINGbutton">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>20</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Styling for the QPushButton within the QWidget with ID &quot;widget&quot; */
QWidget#widget QPushButton {
    background-color: #4d90fe; /* Button background color */
    color: #fff; /* Button text color */
    border: 1px solid #357ae8; /* Button border color */
    border-radius: 10px; /* Rounded corners for buttons */
    font: 14pt &quot;MS Shell Dlg 2&quot;; /* Font size and family */
}

QWidget#widget QPushButton:hover {
    background-color: #357ae8; /* Darker button background on hover */
}
</string>
     </property>
     <property name="text">
      <string>PING</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>866</x>
       <y>10</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget QLabel#label_2 {
    font: 9pt &quot;MS Shell Dlg 2&quot;; /* Font size and family */
    color: #333; /* Text color */
   
}</string>
     </property>
     <property name="text">
      <string>Rajesh K</string>
     </property>
    </widget>
    <widget class="QLabel" name="err_label">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>380</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;color:red</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ServiceStatusButton">
     <property name="geometry">
      <rect>
       <x>338</x>
       <y>20</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Styling for the QPushButton within the QWidget with ID &quot;widget&quot; */
QWidget#widget QPushButton {
    background-color: #4d90fe; /* Button background color */
    color: #fff; /* Button text color */
    border: 1px solid #357ae8; /* Button border color */
    border-radius: 10px; /* Rounded corners for buttons */
    font: 14pt &quot;MS Shell Dlg 2&quot;; /* Font size and family */
}

QWidget#widget QPushButton:hover {
    background-color: #357ae8; /* Darker button background on hover */
}
</string>
     </property>
     <property name="text">
      <string>Service Status</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ServiceStartButton">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>20</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Styling for the QPushButton within the QWidget with ID &quot;widget&quot; */
QWidget#widget QPushButton {
    background-color: #4d90fe; /* Button background color */
    color: #fff; /* Button text color */
    border: 1px solid #357ae8; /* Button border color */
    border-radius: 10px; /* Rounded corners for buttons */
    font: 14pt &quot;MS Shell Dlg 2&quot;; /* Font size and family */
}

QWidget#widget QPushButton:hover {
    background-color: #357ae8; /* Darker button background on hover */
}
</string>
     </property>
     <property name="text">
      <string>Start Service</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>946</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>946</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>822</x>
       <y>30</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>85</y>
       <width>356</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>65</y>
       <width>891</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>95</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Server / Host List</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_servers">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>120</y>
       <width>111</width>
       <height>206</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget {
                background-color: #f5f5f5;
                border: 1px solid #ddd;
                color: #333;
            }

            QListWidget::Item {
                padding: 3px;
            }

            QListWidget::Item:selected {
                background-color: #2ecc71;
                color: #333;
            }

            QListWidget::Item:alternate {
                background-color: #ecf0f1;
            }

            QListWidget::Item:hover {
                background-color: #aaff7f;
            }
</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_failed">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>111</width>
       <height>206</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget {
                background-color: #f5f5f5;
                border: 1px solid #ddd;
                color: #333;
            }

            QListWidget::Item {
                padding: 3px;
            }

            QListWidget::Item:selected {
                background-color: #2ecc71;
                color: #333;
            }

            QListWidget::Item:alternate {
                background-color: #ecf0f1;
            }

            QListWidget::Item:hover {
                background-color: #aaff7f;
            }
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>95</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PING: Failed List</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_passed">
     <property name="geometry">
      <rect>
       <x>235</x>
       <y>120</y>
       <width>111</width>
       <height>206</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget {
                background-color: #f5f5f5;
                border: 1px solid #ddd;
                color: #333;
            }

            QListWidget::Item {
                padding: 3px;
            }

            QListWidget::Item:selected {
                background-color: #2ecc71;
                color: #333;
            }

            QListWidget::Item:alternate {
                background-color: #ecf0f1;
            }

            QListWidget::Item:hover {
                background-color: #aaff7f;
            }
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>435</x>
       <y>95</y>
       <width>66</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Service  List</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_service">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>120</y>
       <width>175</width>
       <height>206</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget {
                background-color: #f5f5f5;
                border: 1px solid #ddd;
                color: #333;
            }

            QListWidget::Item {
                padding: 3px;
            }

            QListWidget::Item:selected {
                background-color: #2ecc71;
                color: #333;
            }

            QListWidget::Item:alternate {
                background-color: #ecf0f1;
            }

            QListWidget::Item:hover {
                background-color: #aaff7f;
            }
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>585</x>
       <y>95</y>
       <width>116</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Service: Running</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_serviceRunning">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>120</y>
       <width>175</width>
       <height>206</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget {
                background-color: #f5f5f5;
                border: 1px solid #ddd;
                color: #333;
            }

            QListWidget::Item {
                padding: 3px;
            }

            QListWidget::Item:selected {
                background-color: #2ecc71;
                color: #333;
            }

            QListWidget::Item:alternate {
                background-color: #ecf0f1;
            }

            QListWidget::Item:hover {
                background-color: #aaff7f;
            }
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>95</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PING: Pass List</string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>347</x>
       <y>95</y>
       <width>16</width>
       <height>241</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>325</y>
       <width>931</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="Logs_text">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>375</y>
       <width>941</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPlainTextEdit {
                background-color: #282c34;
                color: #abb2bf;
                font-family: Consolas, 'Courier New', monospace;
                font-size: 12px;
            }

            QScrollBar:vertical {
                border: 1px solid #2e3036;
                background: #282c34;
                width: 10px;
                margin: 0;
            }

            QScrollBar::handle:vertical {
                background-color: #61afef;
                min-height: 20px;
            }

            QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
                height: 0;
            }

            QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
                background: none;
            }</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>345</y>
       <width>66</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Logs:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_ServiceNOTRunning">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>120</y>
       <width>196</width>
       <height>206</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget {
                background-color: #f5f5f5;
                border: 1px solid #ddd;
                color: #333;
            }

            QListWidget::Item {
                padding: 3px;
            }

            QListWidget::Item:selected {
                background-color: #2ecc71;
                color: #333;
            }

            QListWidget::Item:alternate {
                background-color: #ecf0f1;
            }

            QListWidget::Item:hover {
                background-color: #aaff7f;
            }
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>95</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Service: NOT Running</string>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>110</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_7">
     <property name="geometry">
      <rect>
       <x>385</x>
       <y>110</y>
       <width>561</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_8">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>85</y>
       <width>566</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_9">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>95</y>
       <width>16</width>
       <height>241</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="cmdSaveLogs">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>335</y>
       <width>106</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">Save log</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="cmdSendLogEmail">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>335</y>
       <width>146</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">Send log in email</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="cmdSendReportEmail">
     <property name="geometry">
      <rect>
       <x>775</x>
       <y>335</y>
       <width>161</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">Send Reports in Email</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbPing">
     <property name="geometry">
      <rect>
       <x>575</x>
       <y>348</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ping Report</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckService">
     <property name="geometry">
      <rect>
       <x>665</x>
       <y>348</y>
       <width>96</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Service Report</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLoad">
    <property name="title">
     <string>Load</string>
    </property>
    <addaction name="actionLoad_Server_List"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Service_List"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Email_list"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionPING"/>
    <addaction name="separator"/>
    <addaction name="actionCheck_Service_Status"/>
    <addaction name="separator"/>
    <addaction name="actionStart_Service"/>
    <addaction name="separator"/>
    <addaction name="actionSend_Ping_Report"/>
    <addaction name="separator"/>
    <addaction name="actionSend_Service_Report"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionContact_Me"/>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Close</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuLoad"/>
   <addaction name="menuRun"/>
   <addaction name="menuExit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Server_List">
   <property name="text">
    <string>Load Server List</string>
   </property>
  </action>
  <action name="actionLoad_Service_List">
   <property name="text">
    <string>Load Service List</string>
   </property>
  </action>
  <action name="actionLoad_Email_list">
   <property name="text">
    <string>Load Email List</string>
   </property>
  </action>
  <action name="actionCheck_Service_Status">
   <property name="text">
    <string>Service Status</string>
   </property>
  </action>
  <action name="actionPING">
   <property name="text">
    <string>PING</string>
   </property>
  </action>
  <action name="actionSend_Ping_Report">
   <property name="text">
    <string>Send Ping Report</string>
   </property>
  </action>
  <action name="actionSend_Service_Report">
   <property name="text">
    <string>Send Service Report</string>
   </property>
  </action>
  <action name="actionContact_Me">
   <property name="text">
    <string>Contact Me</string>
   </property>
  </action>
  <action name="actionStart_Service">
   <property name="text">
    <string>Start Service</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>2</number>
  </property>
  <property name="gridDeltaY">
   <number>2</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
